demo/web/src/main/notebooks/Python Data Science/Functions.md

```diff
@@ -0,0 +1,68 @@
+# **Python functions in Deephaven**
```

Comment:
could we make this broader? "Using functions in Deephaven"

- with that said, do we want only Python? both Python and Groovy? and in separate notebooks